src/AutorestSwift/JsonRpc/Server.swift

```diff
@@ -124,7 +123,7 @@ private class Handler: ChannelInboundHandler, RemovableChannelHandler {
     }
 
     public func errorCaught(context: ChannelHandlerContext, error: Error) {
-        SharedLogger.debug("Server Handler errorCaught")
+        SharedLogger.error("Server Handler errorCaught")
```

Comment:
Same comment. Should we log the error contents?

Reply:
Good point.